app/views/application/_flash.html.erb

```diff
@@ -0,0 +1,5 @@
+<% flash.each do |key, value| %>
+  <div class="flash-<%= key %>">
+    <span><%= value %></span>
+  </div>
+<% end %>
```

Comment:
This is great. You can use this same idea for errors, just need to pass in the object that you will be reading the messages from.